pyproject.toml

```
@@ -19,7 +19,8 @@ copilot-helper = "copilot_helper:copilot_helper"
 [tool.poetry.dependencies]
 Jinja2 = "3.1.2"
 PyYAML = "6.0.1"
-boto3 = "^1.26.31"
+#boto3 = "^1.26.31"
+boto3 = "1.28.24"
```

Are we okay to do this?

```suggestion
boto3 = "^1.28.24"
```

---

I'll check - but I think I did that last week because of a breaking change.

---

No - must have imagined that. Made more lenient again.

---

Resolved in https://github.com/uktrade/copilot-tools/pull/173/commits/4aa1e0d70c5ae7017b94222809e67050e1745723.